simplq/src/components/Layout/Routes.jsx

```diff
@@ -5,7 +5,7 @@ import TokenStatusPage from 'components/pages/TokenStatus/TokenStatusPage';
 import AdminPage from 'components/pages/Admin/AdminPage';
 import PageNotFound from 'components/pages/PageNotFound';
 import PopupNotifications from 'components/common/Popup';
-import HomePage from 'components/pages/Home/HomePage';
+import Home from 'components/pages/Home/HomePage';
```

Comment:
I have addressed filename changes in #559. As explained [here](https://github.com/SimplQ/simplQ-frontend/pull/559#pullrequestreview-600687938):

> Wherever the reason for the code smell rename suggestion is the single class export, you should opt to change the name of the class, rather than the filename. The most common reason for this case is that developer (including myself 😊 ) decided to change the file name for some good reason but forgot to change the class name. The second most common is creating files by copy-paste.

Reply:
> I have addressed filename changes in #559. As explained [here](https://github.com/SimplQ/simplQ-frontend/pull/559#pullrequestreview-600687938):
> 
> > Wherever the reason for the code smell rename suggestion is the single class export, you should opt to change the name of the class, rather than the filename. The most common reason for this case is that developer (including myself blush ) decided to change the file name for some good reason but forgot to change the class name. The second most common is creating files by copy-paste.

@mradenovic  should I revert this ?

Reply:
Yes please! You can change the class name in `HomePage.tsx` to match the filename.